tests/unit/test_schema_backup.py

```diff
@@ -2,13 +2,69 @@
 Copyright (c) 2023 Aiven Ltd
 See LICENSE for details
 """
-from karapace.schema_backup import _writer
+from kafka import KafkaConsumer, KafkaProducer
+from kafka.structs import PartitionMetadata
+from karapace import config
+from karapace.config import Config
+from karapace.schema_backup import _consumer, _producer, _writer, PartitionCountError
 from pathlib import Path
+from types import FunctionType
+from typing import AbstractSet, Callable, Type, Union
+from unittest import mock
 
 import pytest
 import sys
 
 
+class TestClients:
+    @staticmethod
+    def _partition_metadata(c: int = 1) -> AbstractSet[PartitionMetadata]:
+        return {PartitionMetadata("topic", i, 0, tuple(), tuple(), None) for i in range(0, c)}
+
+    @pytest.mark.parametrize(
+        "f,t,m",
```

Comment:
I'd like to have longer names for variables and parameters, makes quite a bit easier to read. Applies to other places too.